homeassistant/components/toon/oauth2.py

```diff
@@ -132,4 +132,6 @@ async def _token_request(self, data: dict) -> dict:
 
         resp = await session.post(self.token_url, data=data, headers=headers)
         resp.raise_for_status()
-        return cast(dict, await resp.json())
+        resp_json = cast(dict, await resp.json())
+        resp_json["expires_in"] = float(resp_json["expires_in"])
```

Comment:
If this is the case, it is a bug @ their end.
See OAuth specification: <https://tools.ietf.org/html/rfc6749#appendix-A.14>

Let us add in a comment for this in the code to make that clear why we do this here.

I'll send out a message to Quby, so they could maybe fix it?